lib/modules/dosomething/dosomething_campaign/dosomething_campaign.module

```diff
@@ -761,14 +761,17 @@ function dosomething_campaign_add_shipment_form_vars($node) {
 function dosomething_campaign_is_hot_module(&$vars) {
   $hot_module_enabled = dosomething_helpers_get_variable('node', $vars['nid'], 'hot_module_enabled');
   if ($hot_module_enabled && isset($vars['field_high_season'])) {
+    date_default_timezone_set('EST');
```

Comment:
if we are setting the timezone here, do we also need `dosomething_campaign_set_est_timezone`?